### PR TITLE
Added the Fluka button and removed the Local/Remote buttons in the demo

### PR DESCRIPTION
**Pull request by MatoKnap**

### CODE REVIEW

File: src/WrapperApp/components/InputEditor/InputEditorPanel.tsx [hunk 1]
```diff
@@ -148,17 +148,19 @@ export default function InputEditorPanel({
...
-							color='warning'>
-							Remote
-						</ToggleButton>
+						<>
+							<ToggleButton
+								color='info'
+								value='local'>
+								Local
+							</ToggleButton>
+							<ToggleButton
```
Comment (grzanka) [inline src/WrapperApp/components/InputEditor/InputEditorPanel.tsx:158]: can you remove completely these toogle buttons and use only `Local` way of generating input files ?

File: src/WrapperApp/components/InputEditor/InputEditorPanel.tsx [hunk 2]
```diff
@@ -49,11 +45,8 @@ export default function InputEditorPanel({
 		async (editorJSON: EditorJson): Promise<SimulationInputFiles> => {
 			await addCustomStoppingPowerTableToEditorJSON(editorJSON);
 
+			//Currently there is only one generator
 			switch (generator) {
-				case 'remote':
```
Comment (MatoKnap) [inline src/WrapperApp/components/InputEditor/InputEditorPanel.tsx:53]: Should I also remove:

- the code responsible for selecting a potentially different converter
- the code responsible for sending a request to a remote converter from src/services/ShSimulatorService.tsx

File: src/WrapperApp/components/InputEditor/InputEditorPanel.tsx [hunk 3]
```diff
@@ -62,7 +55,7 @@ export default function InputEditorPanel({
 					throw new Error('Unknown generator: ' + generator);
 			}
 		},
-		[controller.signal, convertJSON, convertToInputFiles, generator, simulator]
+		[controller.signal, convertJSON, generator, simulator]
 	);
```
Comment (MatoKnap) [inline src/WrapperApp/components/InputEditor/InputEditorPanel.tsx:59]: Should I also remove:

- the code responsible for selecting a potentially different converter
- the code responsible for sending a request to a remote converter from src/services/ShSimulatorService.tsx

File: src/WrapperApp/components/InputEditor/InputEditorPanel.tsx (hunk 1 above)
Comment (MatoKnap) [reply src/WrapperApp/components/InputEditor/InputEditorPanel.tsx]: Should I also remove:

- the code responsible for selecting a potentially different converter
- the code responsible for sending a request to a remote converter from src/services/ShSimulatorService.tsx

Comment (grzanka) [reply src/WrapperApp/components/InputEditor/InputEditorPanel.tsx]: Yes, that would be great !

**Review by grzanka — APPROVED** [review]

**Pull request merged**